manifests/pulp.pp

```diff
@@ -97,10 +104,16 @@
   Optional[Stdlib::Absolutepath] $https_cert = undef,
   Optional[Stdlib::Absolutepath] $https_key = undef,
   Optional[Stdlib::Absolutepath] $https_ca_cert = undef,
+  Boolean $expose_mongodb = false,
+  Boolean $expose_mongodb_ipv6 = false,
```

Comment:
Why would this default to false? In which case are you not able to bind to IPv6? Note that in our install Redis already binds to `::1` so not having IPv6 is already a requirement. And if you go that far, why do we need a parameter for it?

Reply:
I think you are saying we can default to exposing on IPv4 and IPv6 in all cases? Is there a case where IPv4 only is required? Or IPv6 only is required?

Reply:
Indeed I am. I think IPv4-only would be required if the kernel has no IPv6 support but we don't support that case anymore. I'm not sure you can actually run the Linux kernel without IPv4 support so binding should always work.

Maybe I'm missing something, but I'd prefer to at least default to true. Another option is:
```puppet
  Variant[Boolean, Array[Stdlib::IP::Address]] $expose_mongodb = false,
```

Then `true` would imply dual stack. If the user needs full control over exact binding, you have it and only need a single parameter.

Reply:
I set the default to true and left it there given this is designed for a specific use case and I think the parameters provide the needed flexibility.